ci/build/build-vscode.sh

```diff
@@ -23,6 +23,9 @@ copy-bin-script() {
   # shellcheck disable=SC2016
   sed -i.bak 's/^ROOT=\(.*\)$/VSROOT=\1\nROOT="$(dirname "$(dirname "$VSROOT")")"/g' "$dest"
   sed -i.bak 's/ROOT\/out/VSROOT\/out/g' "$dest"
+  # We do not want expansion here; this text should make it to the file as-is.
+  # shellcheck disable=SC2016
+  sed -i.bak 's/$ROOT\/node/${CS_NODE_EXEC_PATH:-ROOT\/lib\/node}/g' "$dest"
```

Comment:
My bad, I missed the $ on the root var

```suggestion
  sed -i.bak 's/$ROOT\/node/${CS_NODE_EXEC_PATH:-$ROOT\/lib\/node}/g' "$dest"
```

Reply:
I missed it too! Thank you for catching!